serverless/aws/src/index.ts

```diff
@@ -160,20 +160,12 @@ export const handlerRaw = async (
     const header = await p.getHeader();
 
     if (!tile) {
-      if (!process.env.PUBLIC_HOSTNAME) {
-        return apiResp(
-          501,
-          "PUBLIC_HOSTNAME must be set for TileJSON",
-          false,
-          headers
-        );
-      }
       headers["Content-Type"] = "application/json";
 
       const t = tileJSON(
         header,
         await p.getMetadata(),
-        process.env.PUBLIC_HOSTNAME,
+        process.env.PUBLIC_HOSTNAME || event.headers['x-distribution-domain-name'] || "",
```

Comment:
Prior to this change, this function was returning a 501 if no  public hostname could be resolved. Here, you're returning an empty string.

Besides notifying a user their setup may not be configured correctly, are there any other reason to avoid returning an empty string?

Reply:
It should probably go back to giving a 501 if both the PUBLIC_HOSTNAME and the distribution domain name header are both blank.